src/app/base/components/comment/comment-helper.ts

```diff
@@ -340,6 +346,10 @@ export class CommentDataManager {
     private buildCurrentDataOrder() {
         this.currentDataOrder = [];
         for (var key in this.currentData) {
+            const findCurrentUser = this.allUsers.find(u => u.id == this.currentData[key].created_by);
+            const avatarSelector = (findCurrentUser.firstName[0].charCodeAt(0) + findCurrentUser.lastName[0].charCodeAt(0)) % 5;
+            const avatarUri = "assets/avatars/" + avatarSelector + ".png"
```

Comment:
i think these two lines are used a lot in the app -> maybe we should build a helper function that gets user and returns the URI so in the future when we want to change it we don't need to change it everywhere :)